ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitInformerView.kt

```diff
@@ -100,6 +134,25 @@ class UiKitInformerView @JvmOverloads constructor(
         }
     }
 
+    private fun View.setBackgroundProperties(
+        @DrawableRes drawableRes: Int? = null,
+        drawable: Drawable? = null,
+        @ColorRes tintRes: Int? = null,
+        @ColorInt tintColor: Int? = null
+    ) {
+        background = when {
+            drawableRes != null -> ContextCompat.getDrawable(context, drawableRes)
```

Comment:
context.getDrawable

Reply:
context.resources.getDrawable(drawable, context.theme) along with context.getDrawable(res) are actually deprecated, and there are only 2 ways to avoid warnings:
ResourceCompat.getDrawable(resources, drawable, theme) or ContextCompat.getDrawable(context, drawable)